doc/index.rst

```diff
@@ -1,9 +1,32 @@
-The Plutus Platform
+Plutus Tools Developer Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
+Plutus Tools are a collection of off-chain infrastructure resources. Plutus Tools are built 
+for developers who are authoring web-based applications that implement smart contracts on the 
```

Comment:
```suggestion
for developers who are authoring applications that implement smart contracts on the 
```

Reply:
There is nothing web-specific about the tools.

Reply:
Great to know, thanks!

Reply:
I'm thinking we could say "for developers who are authoring distributed applications that implement smart contracts on the ..." Or are they not necessarily always going to be distributed applications?